src/main/kotlin/io/bitvavo/simulation/engine/order_book/InMemoryOrderBook.kt

```diff
@@ -0,0 +1,63 @@
+package io.bitvavo.simulation.engine.order_book
```

Comment:
I think in java/kotlin people don't use _ in package names, it is usually a single line

Reply:
ok